runatlantis.io/docs/post-workflow-hooks.md

```diff
@@ -45,6 +43,23 @@ repos:
       # ...
 ```
 
+## Customising the Shell
+
+By default, the commands will be run using the 'sh' shell with an argument of '-c'. This
+can be customised using the `shell` and `shellArgs` keys.
+
+Example:
+
+```yaml
+repos:
+    - id: /.*/
+      post_workflow_hooks:
+        - run: infracost output --path=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-*-infracost.json --format=github-comment --out-file=/tmp/infracost-comment.md
```

Comment:
infracost may not be the best example anymore since they are trying to get everyone to move off the infracost cli and move to their free (and easier to setup) github app.

We can probably just use a hello world type example here

Reply:
Done